zebra/zebra_evpn_mh.c

```diff
@@ -3404,13 +3405,14 @@ DEFPY(zebra_evpn_es_id,
 
 		if (esi_str) {
 			if (!str_to_esi(esi_str, &esi)) {
-				vty_out(vty, "%% Malformed ESI\n");
+				vty_out(vty, "%%Malformed ESI name\n");
```

Comment:
Usually we use whitespace after `%%`.

Reply:
IMO, we should use a consistent version in every file. Then we might fix this everywhere?

Reply:
Done.  Please check it, thanks.